src/viewport_manager.ts

```diff
@@ -152,7 +152,7 @@ export class ViewportManager implements Disposable, NeovimRedrawProcessable, Neo
         }
         const viewport = this.gridViewport.get(gridId);
         if (viewport && startLine != viewport?.topline && currentLine == viewport?.line) {
-            this.client.executeLua("vscode.scroll_viewport(...)", [Math.max(startLine, 0), endLine]);
+            this.client.executeLua("vscode.util.scroll_viewport(...)", [Math.max(startLine, 0), endLine]);
```

Comment:
This fails because `util` is not a member of the vscode M export. Should I change this to a `require(vscode.util)`, or is there a better way for me to access the util module? I think the reason this currently works is `_G.vscode = vscode` in `vscode.lua`.

Reply:
`require('vscode.util')` is the normal idiomatic way.

I would drop `_G.vscode = vscode` unless there is a very good reason it's needed. It adds potential for conflicts with other plugins or user config, so better to avoid it if possible.